sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -823,7 +823,7 @@ public override void Dispose()
         /// <param name="entityPath"></param>
         ///
         /// <param name="connection">The AMQP connection to which the link being refreshed is bound to.</param>
-        /// <param name="amqpLink">The AMQO link to refresh authorization for.</param>
+        /// <param name="amqpLink">The AMQP link to refresh authorization for.</param>
```

Comment:
$1 says that I was responsible for this one.....   😛

Reply:
😝